Source/WebCore/platform/ThreadTimers.cpp

```diff
@@ -43,6 +43,11 @@ namespace WebCore {
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(ThreadTimers);
 
+inline CheckedPtr<SharedTimer> ThreadTimers::protectedSharedTimer()
```

Comment:
We usually call this `checkedSharedTimer()` I believe when returning a CheckedPtr.

Reply:
Good point, will fix.